### PR TITLE
Project 2 - Maria Bergman

### PR DESCRIPTION
**Pull request by Ingetora**
I tried to fullfill all of the requirements, blue, red and black. Did not use CSS-grid. I designed the page like this:

Mobile: Content i one column.
iPad: Middle part of site with polaroids and embedded video links, side by side.
Big iPad/small desktop: Middle part side by side, polaroids side by side.
Big desktop: Header and footer widens, but middle part stays 1200px for readability and easy overlook.

I spent a lot of time figuring out flexbox and classes within flexbox and classes. ;-)

### CODE REVIEW

**Comment by JennieDalgren** [comment]
Hi! Do you have a deployed version of this? Please provide a link to the project 🙂

**Comment by Ingetora** [comment]
Hi, Jennie

Thank you for emailing me!

I did provide a link to the Project in Canvas, as it says in the instruction. Did not know there was supposed to be a link at GitHub as well but I will add one from now on, here it is:

https://dazzling-lalande-a35b10.netlify.app/

Do I need to do a new pull request on GitHub?

Med vänlig hälsning Maria
________________________________
Från: Jennie Dalgren <notifications@github.com>
Skickat: den 29 januari 2021 18:56
Till: Technigo/project-news-site <project-news-site@noreply.github.com>
Kopia: Maria Bergman <maria.bergman@outlook.com>; Author <author@noreply.github.com>
Ämne: Re: [Technigo/project-news-site] Project 2 - Maria Bergman (#141)


Hi! Do you have a deployed version of this? Please provide a link to the project 🙂

**Comment by JennieDalgren** [comment]
no need for a new PR, they are updated every time you make a comment, push new changes to your code and similar.
For future projects, add a link to the live project in your readme as written in the example

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,150 @@
...
+
+.news-card {
+  display: flex;
+  flex-direction: row;
+  width: auto;
+  height: 310px;
+  margin-bottom: 20px;
+}
+
+.style {
```
Comment (JennieDalgren) [inline code/style.css:30]: this classname is a bit unclear, could you change it to one thats a bit more descriptive

**Review by JennieDalgren — CHANGES_REQUESTED** [review]
Good job with this project, flexbox is a hard one to wrap your head around.

Make sure to fix indentation in both the html and css file, before I can approve your project.
We know, this is a bit picky but we want you to get into habit as soon as possible to write clean and tidy code.

Take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you need and change this.
We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

Keep up the good work! ⭐️💪🏼

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,76 @@
...
+        <iframe height="250" src="https://www.youtube.com/embed/ljmfzjSW1Ew">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/ljmfzjSW1Ew" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </iframe><br>
+        <article>If you want to get started!
+        </article>
+        </aside>
+    
+        </div>
+        
+        <!-- Footer -->
```
[the remark this answers is not in the document]
Comment (Ingetora) [reply code/index.html:69]: Thank you!

File: code/style.css (hunk 1 above)
Comment (Ingetora) [reply code/style.css]: It's the styling of the cards, so I changed the class to "cards".

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,150 @@
...
+
+main, header, section, aside, footer {
+  margin: 0;
+  padding: 20px;
+  border: none;
+  color: dimgray;
+  text-align: center;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 24px;
+/*     border: 1px solid #000000; */
```
[the remark this answers is not in the document]
Comment (Ingetora) [reply code/style.css]: Off course! I have done a git add, commit and push on the updated code. Thank you for noticing, and thank you so much for my first code review! XD

**Review by JennieDalgren — APPROVED** [review]
👍

**Pull request closed without merging**